maven-surefire-plugin/src/site/apt/examples/junit-platform.apt.vm

```diff
@@ -411,6 +411,42 @@ Using JUnit 5 Platform
 </dependencies>
 +---+
 
+** JUnit5 Suite
+
+  For more information see this
+  {{{https://github.com/apache/maven-surefire/tree/master/surefire-its/src/test/resources/junit5-suite}example}}
+  and the {{{https://junit.org/junit5/docs/current/user-guide/#junit-platform-suite-engine}tutorial}}.
+
++---+
+<dependencies>
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>5.8.2</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.junit.platform</groupId>
+        <artifactId>junit-platform-suite-engine</artifactId>
+        <version>1.8.2</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
```

Comment:
I'm still not sure if we want to write documentations for external products ....  😄 
But from other side we should be consistent with original product 
documentation.
https://junit.org/junit5/docs/current/user-guide/#junit-platform-suite-engine-setup-required-dependencies

There is:

> Required Dependencies
> - junit-platform-suite-api in test scope: artifact containing annotations needed to configure a test suite
> - junit-platform-suite-engine in test runtime scope: implementation of the TestEngine API for declarative test suites

Reply:
@slawekjaranowski 
Slawomir, we have never repeated "their" documentation. We have always declared Surefire configuration in order to run some tests, support some framework, declaring features we support and we made a fix to support these features.
The JUnit documentation should mentioned us as Surefire, we have to do it, we are the one who should understand how the framework works and how it should be used and configured in the Surefire/Failsafe, not opposite.